src/main/java/com/redhat/cloud/policies/app/model/Policy.java

```diff
@@ -86,19 +89,21 @@ public class Policy extends PanacheEntityBase {
             readOnly = true,
             format = "yyyy-MM-dd hh:mm:ss.ddd",
             implementation = String.class)
+    @JsonProperty(access = JsonProperty.Access.READ_ONLY)
```

Comment:
The `read_only` is on the sense of the json-string to/from java class. 
`read_only` here means this property will only be `read` (i.e. when converting from class to json) and `write_only` (not used in this PR) is when converting from json to class (i.e. write the class attribute).

I'm clarifying because is always confusing to me after a while. read/write is not applicable to the json string, but to the class itself.